.mailmap

```diff
@@ -18,6 +18,8 @@
 @endolith <endolith@gmail.com> endolith <endolith@gmail.com>
 @FormerPhysicist <Former@physicist.net> FormerPhysicist <Former@physicist.net>
 @gaulinmp <gaulinmp+git@gmail.com> Mac <gaulinmp+git@gmail.com>
+@h-vetinari <h.vetinari@gmx.com> h-vetinari <h.vetinari@gmx.com>
```

Comment:
Is this one really subtle? It doesn't really hurt to map same-same if they are the same though.

Could just be left over from copy-paste too though, I often do that as a template when I'm working on it.

Reply:
It's on purpose, it maps to the GitHub username, so adds the `@` here - just like the other handles right above it.

Reply:
Ah, I thought we were trying to avoid mapping to the GitHub names to avoid the name-linking that happens in the GitHub release pages for a small subset of the authors that are mapped that way.

I guess the release script handles that now, since we've had this discussion previously.